### PR TITLE
Mixtrack 3 pro controller mapping v1.4 -auto loop mapping adjustment

### PR DESCRIPTION
**Pull request by djsteph**
Adjustments to mapping of Autoloop pads to change default values from 1/8, 1/4, 1/2 and 1 beat loop to 1,2,4 and 8 beat loops

The only file changed is the Numark-Mixtrack-3-scripts.js file. I am unsure if I am doing this pull request correctly. If not, please let me know. 

Thanks
Steph

### CODE REVIEW

File: lib/gmock-1.7.0/gtest/test/gtest-test-part_test (1).cc [hunk 1]
```diff
@@ -0,0 +1,208 @@
+// Copyright 2008 Google Inc.
```
Comment (Be-ing) [inline lib/gmock-1.7.0/gtest/test/gtest-test-part_test (1).cc]: I'm not sure how this happened, but a couple of unrelated files got added in this commit.

Comment (djsteph) [inline lib/gmock-1.7.0/gtest/test/gtest-test-part_test (1).cc]: Hello Be,

As I said, I had a real hard time with Github... Actually, I find this to be the hardest thing to do in contributing. I tried to do a clean fork for this, but it keep bringing me to the old fork I had. Can I simply send you the .js file so that you can integrate it?
Thanks

Comment (Be-ing) [inline lib/gmock-1.7.0/gtest/test/gtest-test-part_test (1).cc]: I think you've confused git branches and GitHub forks. They're not the same thing. Forking creates a clone of the git repository on GitHub's server; branching creates a new place to work within a repository. Generally, the workflow for contributing an update is to create a new branch in your local git repository (which was forked from mixxxdj/mixxx originally), merge the latest changes from mixxxdj/mixxx's master branch into your new branch, then commit your changes and open a pull request.

But yeah, I can make a clean branch and submit a PR for you.

http://xkcd.com/1597/ :P

Comment (djsteph) [inline lib/gmock-1.7.0/gtest/test/gtest-test-part_test (1).cc]: I liked the cartoon.. I did create a fork from Mixxx (
https://github.com/djsteph/mixxx-1) and created a branch (
https://github.com/djsteph/mixxx-1/tree/Mixtrack-3-Pro-v1.4---AutoLoop-mapping)
on my local repository. Where I think I messed up is synchronizing my fork
with the recent change from Mixxx. Just not sure how to do that..

In any event, here are the updated files attached in the zip file

Thanks
Steph

Stéphane Morin
Maison: 450 510-0622
Cellulaire: 514 243-5876

**Comment by djsteph** [comment]
i'll close this request in order to submit a clean one
Steph
